tup.config

```diff
@@ -1,2 +1 @@
-CONFIG_CC32=gcc-x86_64-x86.bat
```

Comment:
It would be very helpful if you squashed commits like this with the ones that introduced the files in the first place. It will be easier for me to test and review.